home/screens/ProjectsScreen.tsx

```diff
@@ -280,40 +286,43 @@ export default class ProjectsScreen extends React.Component {
     );
   };
 
-  _renderRecentHistoryItems = () => {
+  private _renderRecentHistoryItems = () => {
     const extractUsername = manifestUrl => {
-      let username = manifestUrl.match(/@.*?\//)[0];
+      const username = manifestUrl.match(/@.*?\//)[0];
       if (!username) {
         return null;
       } else {
         return username.slice(0, username.length - 1);
       }
     };
 
-    return this.props.recentHistory.map((project, i) => (
-      <SmallProjectCard
-        key={project.manifestUrl}
-        iconUrl={project.manifest && project.manifest.iconUrl}
-        releaseChannel={
-          /* 28/11/17(brentvatne) - we can remove extractReleaseChannel in a couple of months
+    return this.props.recentHistory.map(
+      (project, i) =>
+        project && (
+          <SmallProjectCard
+            key={project.manifestUrl}
+            iconUrl={project.manifest?.iconUrl}
+            releaseChannel={
+              /* 28/11/17(brentvatne) - we can remove extractReleaseChannel in a couple of months
           when project history is unlikely to include any projects with release channels */
```

Comment:
@brentvatne Is this comment this relevant? Or can we remove the releaseChannel prop here ?

Reply:
we can keep the releaseChannel prop but remove the `extractReleaseChannel(project.manifestUrl` part - so this can become `project.manifest?.releaseChannel`

Reply:
Alright thanks, I've removed the extractReleaseChannel code completely 👍